python/lsst/afw/table/tableLib.py

```diff
@@ -1 +1,23 @@
 from __future__ import absolute_import
+
+from .misc import *
+from ._flag import *
+from .keyBase import *
+from ._fieldBase import *
+from .key import *
+from .field import *
+from ._functorKey import *
+from .arrays import *
+from ._aggregates import *
+from .aliasMap import *
+from .schema import *
+from ._schemaMapper import *
+from .baseRecord import *
+from ._baseTable import *
+from .baseColumnView import *
+from .catalog import *
+from ._simple import *
+from ._source import *
+from ._sortedCatalog import *
+
+import lsst.afw.geom.ellipses
```

Comment:
Add the comment:

# In order for some of the module to be imported, lsst.afw.geom.ellipses is required
# Even though the modules themselves are wrapped using ellipses, it must still be
# explicitly imported in python